src/crimson/os/alienstore/alien_store.cc

```diff
@@ -66,7 +66,15 @@ AlienStore::AlienStore(const std::string& path, const ConfigValues& values)
   g_ceph_context = cct.get();
   cct->_conf.set_config_values(values);
   store = std::make_unique<BlueStore>(cct.get(), path);
-  tp = std::make_unique<crimson::os::ThreadPool>(1, 128, seastar::this_shard_id() + 10);
+
+  long cpu_id = 0;
+  if (long nr_cpus = sysconf(_SC_NPROCESSORS_ONLN); nr_cpus != -1) {
+    cpu_id = nr_cpus - 1;
```

Comment:
It seems that alien threads of all `crimson-osd` instances would be pinned to the same core. If so, we might to rework this in the future (single instance inside with cpuset for a container ?) but for testing we don't to worry too much. LGTM!

Reply:
yes, i assume that the "last core" will always be the 2nd core in that container.